cosmosdb/management_endpoint.bal

```diff
@@ -379,14 +389,14 @@ public client class ManagementClient {
         return finalStream;
     }
 
-    # Delete an existing User Defined Function.
+    # Deletes an existing User Defined Function.
     #
     # + databaseId - ID of the database to which the container belongs to
     # + containerId - ID of the container which contains the User Defined Function
     # + userDefinedFunctionid - ID of UDF to delete
```

Comment:
Suggestion: "The ID of the UDF (User-defined Function) to delete"

If "UDF" is a commonly used abbreviation, use it in all the places.